frontend/src/components/message/useMessage.ts

```diff
@@ -0,0 +1,62 @@
+import { KeyboardEvent, useRef, useState } from "react";
+
+const useMessage = () => {
+    const [selected, setSelected] = useState("");
+    const [currentMessage, setCurrentMessage] = useState("");
+    const messageInput = useRef<HTMLInputElement>(null!);
+
+    const input = messageInput.current;
+
+    const sendRequest = () => {
+        //TODO: implement
+    };
+
+    const messageHasChanged = () => input.textContent !== currentMessage;
+
+    const handleAcceptMessage = (event: KeyboardEvent<HTMLInputElement>) => {
+        if (!(event.key === "Enter" || event.key === "Escape")) {
+            return;
```

Comment:
That's unnecessary, if u don't want to return something specific, leave it alone, function/metod return "undefined" default.

Look for statement like this in project, and remove.